tests/integration/expected_test_results/import_systemeu/import_systemeu_export.csv

```diff
@@ -1,6 +1 @@
-code	lc	product_name_fr	quantity	brands	categories	labels	countries	ingredients_text_fr	allergens	nutrition_data_per	nutrition_data_prepared_per	energy-kj_value	energy-kj_unit	energy-kcal_value	energy-kcal_unit	fat_value	fat_unit	saturated-fat_value	saturated-fat_unit	carbohydrates_value	carbohydrates_unit	sugars_value	sugars_unit	fiber_value	fiber_unit	proteins_value	proteins_unit	salt_value	salt_unit	sodium_value	sodium_unit	energy_value	energy_unit	data_sources	obsolete
-3368954957571	fr	Bière blonde d'Abbaye Höellingem , 6,5°, 75cl	0.7500 l	U, U Saveurs		Transformé en France	France	Contient du MALT D'ORGE.	Gluten	100g	100g																								0
-3368954957557	fr	Bière blonde d'Abbaye Höellingem 6,5°, 6x25cl	1.5000 l	U, U Saveurs		Conditionné en France, PEFC	France	Contient du MALT D'ORGE.	Gluten	100g	100g																								0
-3256226087708	fr	Chou frisé, France, la pièce	1 UNITE	U, U Bio	"Aliments et boissons à base de végétaux, Aliments d'origine végétale, Aliments à base de fruits et de légumes, Légumes et dérivés, Légumes"	Cultivé en France	France			100g	100g																								0
-3256224372165	fr	Sirop de cassis bouteille de 70cl	0.7000 l	U	Sirops	Transformé en France	France	"Sucre, eau, jus de fruits à base de concentrés 20% (cassis 18%, sureau), acidifiant : acide citrique, arôme naturel de cassis."		100g	100g	0	kJ	0	kcal	0.5	g	0.1	g	78	g	77	g	0.5	g	0.5	g	0.02	g	0.008	g	0	kJ		0
-3256222240022	fr	Sirop de recette gourmande cerise/mûre bouteille de 70cl	0.7000 l	U	Sirops	Transformé en France	France	"Sucre, eau, jus de fruits à base de concentrés 15,8 % (cerise 6%, sureau, mûre 4,1%), arômes naturels, acidifiant : acide citrique."		100g	100g	0	kJ	0	kcal	0.5	g	0.1	g	79	g	78	g	0.5	g	0.5	g	0.02	g	0.008	g	0	kJ		0
+data_sources	obsolete
```

Comment:
Can you run make test-int test='import_systemeu.t :: --update'  and check the output to see if there are error messages?